src/components/common/Header.js

```diff
@@ -30,6 +45,28 @@ function Header() {
         <Navbar.Toggle aria-controls='responsive-navbar-nav' />
         <Navbar.Collapse id='responsive-navbar-nav'>
           <Nav className='ms-auto'>
+            <NavDropdown
+              title={
+                <div className='bell-icon-wrapper'>
+                  <FontAwesomeIcon icon={faBell} size='lg' />
+                  {notifications?.notifications?.filter(
```

Comment:
NB: Makes it easier to understand.
```suggestion
                  {data?.notifications?.filter(
```